polygeist/lib/Conversion/PolygeistToLLVM/PolygeistToLLVM.cpp

```diff
@@ -1496,6 +1501,34 @@ struct GPUReturnOpLowering : public ConvertOpToLLVMPattern<gpu::ReturnOp> {
   }
 };
 
+struct EraseSPIRVBuiltinPattern
+    : public OpRewritePattern<spirv::GlobalVariableOp> {
+  using OpRewritePattern<spirv::GlobalVariableOp>::OpRewritePattern;
+
+  LogicalResult matchAndRewrite(spirv::GlobalVariableOp global,
+                                PatternRewriter &rewriter) const final {
+    auto *parent = global->getParentWithTrait<OpTrait::SymbolTable>();
+    if (!parent)
+      return failure();
+    const auto name = global.getName();
+    auto &region = parent->getRegion(0);
+    bool alreadyDefined =
+        std::any_of(region.begin(), region.end(), [&](auto &block) {
+          return std::any_of(block.begin(), block.end(), [&](auto &op) {
```

Comment:
```suggestion
    // SymbolTable operations are single-region operations.
    auto &block = parent->getRegion(0).front();
    bool alreadyDefined =
          std::any_of(block.begin(), block.end(), [&](auto &op) {
```

Reply:
This is not exactly the same, as nothing prevents a symbol table to have a region with multiple blocks.

Reply:
ok.